pkg/teleporter/relayer.go

```diff
@@ -340,31 +359,120 @@ func createRelayerConfig(
 		DestinationBlockchains: []*config.DestinationBlockchain{},
 		MetricsPort:            constants.AWMRelayerMetricsPort,
 	}
+	return saveRelayerConfig(awmRelayerConfig, relayerConfigPath)
 }
 
-func addChainToRelayerConfig(
+func AddSourceAndDestinationToRelayerConfig(
```

Comment:
also not clear for me why we need this func as we should just use `AddSourceToRelayerConfig` and `AddDestinationToRelayerConfig` imho. also pls ignore this comment if disagree

Reply:
ok this is the most common case, but the other two are also needed. a SDK user would like all of them